Python/pywarpx/picmi.py

```diff
@@ -2457,7 +2458,7 @@ def diagnostic_initialize_inputs(self):
                     variables.add('Bx')
                     variables.add('By')
                     variables.add('Bz')
-                elif dataname in ['ux', 'uy', 'uz', 'Ex', 'Ey', 'Ez', 'Bx', 'By', 'Bz']:
+                elif dataname in ['x', 'y', 'z', 'ux', 'uy', 'uz', 'Ex', 'Ey', 'Ez', 'Bx', 'By', 'Bz', 'Er', 'Et', 'Br', 'Bt']:
```

Comment:
What about the current variables `j`, though?
I don't quite see why they are missing.